src/utils/safenet.ts

```diff
@@ -31,6 +43,7 @@ const convertSafenetBalanceToSafeClientGatewayBalance = (
       balance,
       fiatBalance: ((parseInt(balance) * 1) / 10 ** decimals).toString(),
       fiatConversion: '1.00',
+      safenet: true,
```

Comment:
very minor nitpick: I would call it `isSafenet` or `safenetFlag` or `hasSafenetFlag` just to make it clear it's a boolean to flag a Safenet unified balance asset.